### PR TITLE
VIH-10054 Handle editing hearings with expired case or hearing types

### PR DESCRIPTION
**Pull request by shaed-parkar**
### JIRA link (if applicable) ###

VIH-10054

### Change description ###

Handle editing hearings with expired case or hearing types

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/vh-admin-web/values.dev.template.yaml [hunk 1]
```diff
@@ -8,4 +8,5 @@ java:
     AZUREAD__POSTLOGOUTREDIRECTURI: https://${SERVICE_FQDN}/logout
     AZUREAD__REDIRECTURI: https://${SERVICE_FQDN}/home
     DOM1__POSTLOGOUTREDIRECTURI: https://${SERVICE_FQDN}/logout
-    DOM1__REDIRECTURI: https://${SERVICE_FQDN}/home
\ No newline at end of file
+    DOM1__REDIRECTURI: https://${SERVICE_FQDN}/home
+    VHSERVICES__BOOKINGSAPIURL: https://vh-bookings-api-pr-694.dev.platform.hmcts.net/
```
Comment (shaed-parkar) [inline charts/vh-admin-web/values.dev.template.yaml:12]: reminder to remove

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/create-hearing/create-hearing.component.html [hunk 2]
```diff
@@ -64,23 +64,44 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">Please complete
 
   <div *ngIf="availableCaseTypes.length > 1" [ngClass]="caseTypeInvalid ? 'govuk-form-group--error' : 'govuk-form-group'">
     <label class="govuk-label govuk-!-width-one-half" for="caseType"> Case type </label>
-    <select class="govuk-select govuk-!-width-one-half" id="caseType" formControlName="caseType">
+    <!--suppress XmlDuplicatedId -->
+    <select *ngIf="!this.editMode" class="govuk-select govuk-!-width-one-half" id="caseType" formControlName="caseType">
```
Comment (oliver-scott) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/booking/create-hearing/create-hearing.component.html:68]: shouldn't this be the other way round? If in edit mode, show the dropdown, else show a text box?

File: AdminWebsite/AdminWebsite/packages.lock.json [hunk 3]
```diff
@@ -4,9 +4,9 @@
     "net6.0": {
       "BookingsApi.Client": {
         "type": "Direct",
-        "requested": "[1.46.1, )",
-        "resolved": "1.46.1",
-        "contentHash": "lmopaFe7t5X/6eF0mVNpy0+PrBVRbjOgcuc1Wh39qFAdyXzvxjrDnN9n1QI1hrBTqCQt8ameOisr5jveGAQz0Q==",
+        "requested": "[1.46.2-pr-0694-0002, )",
```
Comment (oliver-scott) [inline AdminWebsite/AdminWebsite/packages.lock.json:7]: just a reminder to update this before merging

**Review by oliver-scott — COMMENTED** [review]
left a couple of comments, looks good otherwise

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/create-hearing/create-hearing.component.html (hunk 2 above)
Comment (shaed-parkar) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/booking/create-hearing/create-hearing.component.html:68]: I think the `EditMode` property might be misleading. This means edit an already booked hearing. You can only choose these values once upon booking, not after, so the drop down is available whilst you're making the booking, not when you're editing

Comment (oliver-scott) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/booking/create-hearing/create-hearing.component.html:68]: ah right ok that makes sense. Cheers

**Review by oliver-scott — APPROVED** [review]

**Review by phildebrah — APPROVED** [review]
Tested and approve

**Pull request merged**